File: c++/11508.cpp
#include <algorithm>
#include <iostream>
#include <vector>

using namespace std;

vector<int> arr;

int main() {
  freopen("example.txt", "r", stdin);

  int N;
  cin >> N;

  while (N--) {
    int a;
    cin >> a;
    arr.push_back(a);
  }

  sort(arr.begin(), arr.end(), greater<>());

  int ans = 0;

  for (int i = 0; i < arr.size(); i++) {
    if (i % 3 == 2) continue;
    ans += arr[i];
  }

  cout << ans;

  return 0;
}

File: js/11508.js
const fs = require('fs');

const [N, ...arr] = fs
  .readFileSync('example.txt')
  .toString()
  .trim()
  .split('\n')
  .map(Number);

const result = arr
  .sort((a, b) => b - a)
  .map((e, i) => (i % 3 === 2 ? 0 : e))
  .reduce((a, b) => a + b);

console.log(result);
